app/assets/javascripts/discourse/app/templates/components/categories-boxes-with-topics.hbs

```diff
@@ -5,7 +5,7 @@
         <a href={{c.url}}>
           {{#unless c.isMuted}}
             {{#if c.uploaded_logo.url}}
-              <CdnImg @src={{c.uploaded_logo.url}} @class="logo" @width={{c.uploaded_logo.width}} @height={{c.uploaded_logo.height}} @alt="" />
+              <CategoryLogo @category={{c}} @class="logo" />
```

Comment:
Minor nitpick, but it doesn't look like the "logo" class is being used, we should remove it. (It's too generic anyway, might clash with other, unrelated styles.)